bazel/repository_locations.bzl

```diff
@@ -823,15 +861,15 @@ REPOSITORY_LOCATIONS_SPEC = dict(
             "envoy.filters.network.wasm",
             "envoy.stat_sinks.wasm",
         ],
-        release_date = "2020-10-22",
+        release_date = "2020-11-10",
```

Comment:
Nit: it looks that you accidentally updated release date of Proxy-Wasm C++ SDK instead of host.

Reply:
oh, thanks!